contracts/src/bridge/Inbox.sol

```diff
@@ -354,7 +354,7 @@ contract Inbox is DelegateCallAware, PausableUpgradeable, IInbox {
     ) public payable virtual override whenNotPaused returns (uint256) {
         // gas price and limit of 1 should never be a valid input, so instead they are used as
         // magic values to trigger a revert in eth calls that surface data without requiring a tx trace
-        if (gasLimit == 1 && maxFeePerGas == 1)
+        if (gasLimit == 1 || maxFeePerGas == 1)
```

Comment:
Why do we need to have both options here? Isnt being able to manipulate one variable enough?

Reply:
hm i was thinking if some L1 contract wants to actually use one of these variable in their message, this allows for more flexibility.
For example, if the L1 token gateway wants to send the user provided gasLimit as information to the L2 gateway, then it can still hit the RetryableData revert without changing the intended L1 to L2 message.